src/app/components/MessageBanner/index.styles.tsx

```diff
@@ -0,0 +1,164 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '../../utilities/pixelsToRem';
+
+const styles = {
+  container: () =>
+    css({
+      paddingTop: '2rem',
+      paddingBottom: '2rem',
+      border: '0.1875rem solid transparent',
+    }),
+  card: ({ mq }: Theme) =>
+    css({
+      height: 'auto',
+      background:
+        'radial-gradient(circle at 20% 90%, #A20219, #180109 60%, #500115 90%)',
+      paddingLeft: '1rem',
+      paddingRight: '1rem',
+      [mq.GROUP_3_ONLY]: {
+        background:
+          'linear-gradient(-120deg, #A20219 0%, #180109 54%, #180109 90%)',
+      },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        background:
+          'linear-gradient(-120deg, #A20219 0%, #180109 54%, #180109 90%)',
+      },
+    }),
+  textWrap: ({ mq }: Theme) =>
+    css({
+      [mq.GROUP_3_ONLY]: {
+        width: '66%',
+      },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        width: '75%',
+      },
+    }),
+  heading: ({ palette }: Theme) =>
+    css({
+      paddingTop: '1.5rem',
+      paddingBottom: '0.5rem',
+      color: palette.WHITE,
+    }),
+  paragraph: ({ palette }: Theme) =>
+    css({
+      paddingBottom: '1rem',
+      color: palette.WHITE,
+    }),
+  imageLtr: ({ mq }: Theme) =>
+    css({
+      maxWidth: '184px',
+      [mq.GROUP_3_ONLY]: {
+        maxWidth: '224px',
+        bottom: 0,
+        right: 0,
+        position: 'absolute',
+      },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        maxWidth: '224px',
+        bottom: 0,
+        right: 0,
+        position: 'absolute',
+      },
+      img: { objectPosition: 'top' },
+    }),
+  imageRtl: ({ mq }: Theme) =>
+    css({
+      maxWidth: '184px',
+      [mq.GROUP_3_ONLY]: {
+        maxWidth: '224px',
+        bottom: 0,
+        left: 0,
+        position: 'absolute',
+      },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        maxWidth: '224px',
+        bottom: 0,
+        left: 0,
+        position: 'absolute',
+      },
+      img: { objectPosition: 'top' },
+    }),
+  linkBackground: ({ mq, palette }: Theme) =>
+    css({
+      padding: '1rem',
+      backgroundColor: palette.WHITE,
+      margin: '0 1rem 1rem 1rem',
+      width: '100%',
+      textDecoration: 'none',
+      '&:hover': {
+        backgroundColor: '#F6F6F6',
+        textDecoration: 'underline',
+        color: palette.BLACK,
+      },
+      '&:focus': {
+        backgroundColor: '#F6F6F6',
+        outline: `${pixelsToRem(3)}rem solid ${palette.WHITE}`,
```

Comment:
Please remove the focus styles other than the background colour, text decoration and colour, then we can pull in @Isabella-Mitchell other focus work

Reply:
This is a tricky one - we only have designs for the new focus style, and we agreed within the team that we would wait until the focus work was live before we put the message banner live - meaning that we don't need to change the designs, and can leave the focus styling as is.

Reply:
I've removed the focus styles other than the background colour, text decoration and colour at Catharine's request and new focus work can be applied at a later date when ready. 😄